--- functions/auth.php	2008-08-17 05:49:09.000000000 +0200
+++ functions/auth.php.oden	2008-09-29 19:40:19.000000000 +0200
@@ -279,7 +279,7 @@
  *
  * @since 1.4.16
  *
- */
+ * this breaks older installations. disable it for now.
 function sqauth_save_password($pass) {
     sqgetGlobalVar('base_uri',    $base_uri,   SQ_SESSION);
 
@@ -289,6 +289,7 @@
     sqsetcookie('key', $key, false, $base_uri);
     return $key;
 }
+ */
 
 /**
  * Fillin user and password based on SMTP auth settings.
